### PR TITLE
[Testing] More Appium actions

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

Added more UITest extension methods to identify the state of an element:
- `bool IsSelected(this IUIElement element);`
- `bool IsDisplayed(this IUIElement element);`
- `bool IsEnabled(this IUIElement element);`

### CODE REVIEW

File: src/TestUtils/src/UITest.Appium/HelperExtensions.cs [hunk 1]
```diff
@@ -104,6 +104,66 @@ public static Rectangle GetRect(this IUIElement element)
...
+			{
+				{ "element", element },
+			});
+
+			if (response?.Value != null)
+			{
+				return (bool)response.Value;
+			}
+
+			throw new InvalidOperationException($"Could not get Displayed of element");
```
Comment (jfversluis) [inline src/TestUtils/src/UITest.Appium/HelperExtensions.cs:164]:
```suggestion
			throw new InvalidOperationException($"Could not get Enabled of element");
```

File: src/TestUtils/src/UITest.Appium/HelperExtensions.cs [hunk 2]
```diff
@@ -1159,14 +1228,45 @@ public static void Lock(this IApp app)
 		/// <exception cref="InvalidOperationException">Unlock is only supported on <see cref="AppiumAndroidApp"/>.</exception>
 		public static void Unlock(this IApp app)
 		{
-			if (app is not AppiumAndroidApp)
+			if (app is not AppiumAndroidApp && app is not AppiumIOSApp)
 			{
-				throw new InvalidOperationException($"Unlock is only supported on AppiumAndroidApp");
+				throw new InvalidOperationException($"Unlock is only supported");
```
Comment (jfversluis) [inline src/TestUtils/src/UITest.Appium/HelperExtensions.cs:1233]:
```suggestion
				throw new InvalidOperationException($"Unlock is only supported on AppiumAndroidApp and AppiumIOSApp");
```

File: src/TestUtils/src/UITest.Appium/HelperExtensions.cs [hunk 3]
```diff
@@ -1159,14 +1228,45 @@ public static void Lock(this IApp app)
...
 
+		/// <summary>
+		/// Check whether the device is locked or not.
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		public static bool IsLocked(this IApp app)
+		{
+			if (app is not AppiumAndroidApp && app is not AppiumIOSApp)
+			{
+				throw new InvalidOperationException($"Unlock is only supported");
```
Comment (jfversluis) [inline src/TestUtils/src/UITest.Appium/HelperExtensions.cs:1247]:
```suggestion
				throw new InvalidOperationException($"IsLocked is only supported on AppiumAndroidApp and AppiumIOSApp");
```

File: src/TestUtils/src/UITest.Appium/Actions/AppiumGeneralActions.cs [hunk 4]
```diff
@@ -59,5 +68,53 @@ CommandResponse GetAttribute(IDictionary<string, object> parameters)
...
+			return CommandResponse.FailedEmptyResponse;
+		}
+
+		CommandResponse GetDisplayed(IDictionary<string, object> parameters)
+		{
+			var element = parameters["element"];
+
+			if (element is AppiumElement appiumElement)
+			{
+				return new CommandResponse(appiumElement, CommandResponseResult.Success);
```
Comment (jfversluis) [inline src/TestUtils/src/UITest.Appium/Actions/AppiumGeneralActions.cs:94]: Is this for older Appium versions? This doesn't seem to check anything specific to displayed, does it? The same of the other ones in here: Enabled, etc.

**Review by jfversluis — CHANGES_REQUESTED** [review]
Couple of small things!

File: src/TestUtils/src/UITest.Appium/Actions/AppiumGeneralActions.cs (hunk 4 above)
Comment (jsuarezruiz) [reply src/TestUtils/src/UITest.Appium/Actions/AppiumGeneralActions.cs:94]: Yes, in all our ui tests using Appium we must have access to the used driver. It a fallback.

**Review by jfversluis — APPROVED** [review]

**Pull request merged**